src/reanimated2/hook/useAnimatedReaction.ts

```diff
@@ -1,33 +1,39 @@
 'use strict';
 import { useEffect } from 'react';
-import type { __BasicWorkletFunction, __WorkletFunction } from '../commonTypes';
+import type { WorkletFunction } from '../commonTypes';
 import { startMapper, stopMapper } from '../core';
 import type { DependencyList } from './commonTypes';
 import { useSharedValue } from './useSharedValue';
 import { shouldBeUseWeb } from '../PlatformChecker';
 
-// ts-prune-ignore-next This will be used in the next PRs in this series.
-export interface AnimatedReactionWorkletFunction<T> extends __WorkletFunction {
-  (prepared: T, previous: T | null): void;
-}
 /**
  * @param prepare - worklet used for data preparation for the second parameter
  * @param react - worklet which takes data prepared by the one in the first parameter and performs certain actions
  * the first worklet defines the inputs, in other words on which shared values change will it be called.
  * the second one can modify any shared values but those which are mentioned in the first worklet. Beware of that, because this may result in endless loop and high cpu usage.
  */
-export function useAnimatedReaction<T>(
-  prepare: __BasicWorkletFunction<T>,
-  react: AnimatedReactionWorkletFunction<T>,
+// @ts-expect-error This overload is required by our API.
```

Comment:
Could you improve this comment please? I would like it to answer the following questions:
1. What is the error we disable
2. Why is it required?

Reply:
I will add explanation to the mother PR.